infrastructure/Taskfile.yml

```diff
@@ -337,7 +337,7 @@ tasks:
         the core and must be available to ssh eg via "eval $(ssh-agent); ssh-add"
       vars:
         ssh_loadbalancer_ip:
-          sh: kubectl get -o jsonpath='{.status.loadBalancer.ingress[0].ip}' -n lagoon-core service lagoon-core-ssh
+          sh: task cluster:auth && kubectl get -o jsonpath='{.status.loadBalancer.ingress[0].ip}' -n lagoon-core service lagoon-core-ssh
```

Comment:
I think you can add `task cluster:auth ` as a dependency to this task and it would run before this task gets executed. Wouldn't it be more elegant?

Reply:
I thought so, too, and so did whoever wrote a lot of the taskfile - but vars are executed *before* deps!

Reply:
Shoot 😅